### PR TITLE
Allow Django-4.2

### PR DESCRIPTION
**Pull request by charleswolfe**

### CODE REVIEW

**Comment by charleswolfe** [comment]
@microsoft-github-policy-service agree [company=default]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Microsoft"

**Comment by charleswolfe** [comment]
@microsoft-github-policy-service agree

File: setup.py [hunk 1]
```diff
@@ -39,7 +40,7 @@
     license='BSD',
     packages=find_packages(),
     install_requires=[
-        'django>=3.2,<4.2',
+        'django>=3.2,<4.3',
```
Comment (medbenmakhlouf) [inline setup.py:43]: I think the condition here should be `'django>=3.2,<5.0'` instead `'django>=3.2,<4.3'`. Django always have 2 minor releases after a major release.

**Comment by mShan0** [comment]
Hi @charleswolfe, thanks for the PR. If you run Tox against Django 4.2, you'll be able to see that 4.2 caused many test failures so we actually have our own working branch to support Django 4.2. You can try the `django42` branch out in the meantime if you need 4.2 immediately. Otherwise, our release is scheduled for May.

**Pull request closed without merging**